types/types.go

```diff
@@ -357,12 +357,12 @@ type ThrottleDevice struct {
 // ShellCommand is a string or list of string args.
 //
 // When marshaled to YAML, nil command fields will be omitted if `omitempty`
-// is specified as a struct tag. Explicitly empty commands (i.e. `[]` or `''`)
+// is specified as a struct tag. Explicitly empty commands (i.e. `[]` or `”`)
```

Comment:
The fun quotes are back 😂

Reply:
This is one of my favorite running gags of the moment 😂

Reply:
```suggestion
// is specified as a struct tag. Explicitly empty commands (i.e. `[]` or `''`)
```